ci-operator/step-registry/openshift/upgrade/qe/sanity/openshift-upgrade-qe-sanity-chain.yaml

```diff
@@ -1,6 +1,7 @@
 chain:
   as: openshift-upgrade-qe-sanity
   steps:
+  - ref: cucushift-upgrade-checktarget
```

Comment:
Hmm, if it's exit here, then no following steps. where to add dedicated OTA case for the test?

Reply:
Do we really need to run OTA cases in case this is a not recommended payload?

Reply:
yes, i think so. there is test case for this checkpoint.

Reply:
cc @shellyyang1989 @jianlinliu for confirm

Reply:
This is to resolve the regular upgrade ci issue to exit in advance. Now we have a dedicated OTA ci yaml, when we need to automate the OTA test case, we can organize a dedicated step chains in a test job in OTA dedicated ci yaml.

Reply:
Yes, agree. so we can keep current step orders